integration/vite-define-route-test.ts

```diff
@@ -0,0 +1,254 @@
+import { test, expect } from "@playwright/test";
+
+import { build, createProject } from "./helpers/vite";
+import dedent from "dedent";
+import stripAnsi from "strip-ansi";
+
+// TODO: passing test for object literal
+// TODO: passing test for `defineRoute`
```

Comment:
I'll add these tests in a follow-up PR since passing tests will require updates to how the route manifest is created.